src/js/components/DataTable/DataTable.js

```diff
@@ -390,14 +442,24 @@ const DataTable = ({
               cellProps={cellProps.body}
               columns={columns}
               data={!paginate ? adjustedData : items}
-              onMore={onMore}
+              onMore={onUpdate ? () => {
+                if (adjustedData.length === limit) {
+                  onUpdate({
+                    expanded: [],
```

Comment:
I'm wondering if we should even include `expanded` when we aren't doing any grouping?

Reply:
removed